azure-pipelines.yml

```diff
@@ -77,6 +75,18 @@ jobs:
         PYTEST_VERSION: '*'
         JOBLIB_VERSION: '*'
         COVERAGE: 'true'
+      pylatest_conda_mkl_no_openmp:
+        DISTRIB: 'conda'
+        PYTHON_VERSION: '*'
+        INSTALL_MKL: 'true'
+        NUMPY_VERSION: '*'
+        SCIPY_VERSION: '*'
+        CYTHON_VERSION: '*'
+        PILLOW_VERSION: '*'
+        PYTEST_VERSION: '*'
+        JOBLIB_VERSION: '*'
+        COVERAGE: 'true'
+        NO_OPENMP: 'true'
```

Comment:
Maybe this env var could be made more specific: `SKLEARN_TEST_NO_OPENMP` to avoid any unintended side-effect caused by an overly generic name.